views/create/create.handlebars

```diff
@@ -67,18 +67,18 @@
 
                 <div class="card-block">
                     <div class='card fieldSeperator'>
-                      <div class='card-header'>
-							<h5>Fields on the card:</h5>
-							<small>Select up to 7 fields.</small>
-						</div>
-					</div>
+                        <div class='card-header'>
+                            <h5>Fields on the card:</h5>
+                            <small>Select up to 7 fields.</small>
+                        </div>
+                    </div>
                     <div id='fields'>
-						<div id='seperator' class='fieldSeperator card'>
-							<div class='card-header'>
-								<h5>Fields not on the card:</h5>
-								<small>Unlimited.</small>
-							</div>
-						</div>
+                        <div id='seperator' class='fieldSeperator card'>
+                            <div class='card-header'>
+                                <h5>Fields not on the card:</h5>
+                                <small>Unlimited.</small>
```

Comment:
Do you need to explicitly say 'unlimited'? I think you leave it out.
Also maybe "▼ Fields not on the card" looks nicer with the arrow.